template.conf

```diff
@@ -18,8 +18,8 @@
 
     {% if options.https %}
       ssl_protocols TLSv1.1 TLSv1.2;
-      ssl_certificate {{ settings.conf }}/../certs/{{ site.shortname }}.cer;
-      ssl_certificate_key {{ settings.conf }}/../certs/{{ site.shortname }}.key;
+      ssl_certificate /etc/letsencrypt/live/{{ site.host }}/fullchain.pem;
```

Comment:
Do these letsencrypt changes have something to do with the websocket functionality? they should probably be reverted.

Reply:
nothing to do with websocket, i fixed it

Reply:
Thanks. There's the additional lines 21 and 22 with the same change. @boutell will be able to give a proper review of the core changes with that reverted.